docs/jvs-apis.md

```diff
@@ -0,0 +1,65 @@
+# JVS APIs
```

Comment:
As we discussed offline, let's make it a API spec doc.

For Justification API, we can reference the grpc service definition at https://github.com/abcxyz/jvs/blob/main/protos/v0/jvs_service.proto

For public key API, we can reference JWKs definition https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-key-sets

For cert rotate API, AFAIK it's empty http call (triggered by cloud scheduler). We should describe its behavior.

Reply:
It's OK if you want to split it up to be per-API. We can do the following for each API:

1. Spec
2. Setup knobs (env vars, flags, etc.)

Reply:
done